optimum/bettertransformer/models/__init__.py

```diff
@@ -114,6 +116,8 @@ class BetterTransformerManager:
         "clip": ["text_model"],
         # blip-2's Q-former and vision model should not be identified as the last layers of the model
         "blip-2": ["qformer.encoder.layer", "vision_model.encoder.layers"],
+        # encodec should not be identified as the last layer
+        "bark": ["codec_model.encoder.layers"],
```

Comment:
I forgot who added blip-2, but what's the link between this dict and not being identified as last layer?

Reply:
The comment is not the good one, I should have said (and I will correct):
`bark.codec_model.encoder is not supported in BetterTransformer`

Without it, the transform doesn't work.

Reply:
```suggestion
        # bark.codec_model.encoder is not supported in BetterTransformer
        "bark": ["codec_model.encoder.layers"],
```